tools/importer/import.js

```diff
@@ -74,7 +74,7 @@ const linkToHlxPage = (main, document, url) => {
     if (new RegExp('^(https?:)?//').test(link.href)) {
       // leave links with domains as is
     } else if (link.href.startsWith('/')) {
-      const newUrl = new URL(link.href, 'https://main--vg-volvotrucks-us--hlxsites.hlx.page');
+      const newUrl = new URL(link.href, 'https://main--vg-volvotrucks-us-rd--netcentric.hlx.page');
```

Comment:
@cogniSyb 
@manuel-vara 
Do we have some steps before prod to change it back to the `https://main--vg-volvotrucks-us--hlxsites.hlx.page`?